androdeb

```diff
@@ -53,14 +54,45 @@ case $key in
 esac
 done
 
-if [ ! -z "$PULL" ]; then
+##########################################################
+#  GIT-PULL
+##########################################################
+
```

Comment:
Lose these extra new lines, and comments. Lets keep it as small a change as possible. The code is self-documenting.